lib/tasks/bundlers/generateFlexChangesBundle.js

```diff
@@ -68,9 +69,10 @@ module.exports = async function({workspace, taskUtil, options: {namespace}}) {
 	const allResources = await workspace.byGlob(
 		`${pathPrefix}/changes/*.{change,variant,ctrl_variant,ctrl_variant_change,ctrl_variant_management_change}`);
 	if (allResources.length > 0) {
-		const version = await readManifestMinUI5Version();
+		const MIN_UI5_VERION_1_73 = new Version("1.73.0");
+		const version = new Version(await readManifestMinUI5Version());
 		let hasFlexBundleVersion = false;
-		if (parseFloat(version) >= 1.73) {
+		if (version.compare(MIN_UI5_VERION_1_73) >= 0) {
```

Comment:
Just my two cents: This is a pretty odd way of using the semver API, that I haven't seen so far and also couldn't find in the documentation at https://github.com/npm/node-semver#readme

IMHO `version.compare(version, "1.73.0") >= 0` would be sufficient and does not involve the creation of `Version` instances.

But I guess your solution works too and since it's your code we are totally fine with merging this as is. **Please let us know soon in case you want to change this. Otherwise we will proceed with releasing this PR.**

Reply:
Edit: I meant `semver.compare`, not `version.compare`

Reply:
i just change it

Reply:
Isn't it necessary to `semver.coerce` the version from the manifest first? I recently worked with `semver` for the first time and was puzzled that it doesn't do this automatically, at least not with the convenience comparison methods (`semver.lt` etc.).

Reply:
Yes, it is. As the manifest.json schema doesn't validate the "minUI5Version" properly (just type "string"), it is likely that the patch version is missing in some cases (e.g. `1.100`). This should also be covered in the tests.

Reply:
Well, even out `manifestCreator` just uses major.minor without the patch version.

Reply:
...which totally makes sense to me as the patch version doesn't play a role wrt. APIs. It's just that `semver` doesn't handle such short versions with a call to `semver.coerce`.